frontend/src/routes/(authenticated)/admin/AdminProjects.svelte

```diff
@@ -31,12 +32,14 @@
       (fromUrl.searchParams.get(key) ?? filterDefaults?.[key])?.toString() !== $filters?.[key]?.toString());
   });
 
-  let filteredProjects: ProjectItem[] = [];
+  let allProjects: ProjectItemWithDraftStatus[] = [];
+  let filteredProjects: ProjectItemWithDraftStatus[] = [];
   let limitResults = true;
   let hasActiveFilter = false;
   let lastLoadUsedActiveFilter = false;
   $: if (!$loading) lastLoadUsedActiveFilter = hasActiveFilter;
-  $: filteredProjects = filterProjects(projects, $filters);
+  $: allProjects = [...projects.map(p => ({ ...p, isDraft: false })), ...draftProjects.map(p => ({ ...p, isDraft: true }))];
```

Comment:
This will put the draft projects at the bottom. Presumably they should share the "create_date" sorting applied to the non drafts.

Reply:
Since they're coming from separate GQL queries, that might be difficult as they don't share enough structure to put them into the same query (which was the first approach I tried). But it would make way more sense to swap these around so that the draft projects are at the *top*. I'll implement that tomorrow and see how it looks.

Reply:
Done in commit 1bd0c86a4dced977ea603288b8b4d4e6a9dbe33a.